### PR TITLE
Adds the missing docs on webhooks.

### PR DESCRIPTION
**Pull request by hydroid7**
**- Summary**
I had to dig into the source to get webhooks working. While it was fun, it should be optional.

**- Test plan**
Just a small documentation improvement.

**- Description for the changelog**
Improved documentation on webhooks.

**- A picture of a cute animal (not mandatory but encouraged)**
:joy_cat:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -289,6 +289,27 @@ Default Content (if template is unavailable):
 <p><a href="{{ .ConfirmationURL }}">Change Email</a></p>
 ```
 
+`WEBHOOK_URL` - `string`
+
+Url of the webhook listener server. This will called when events like  `validate`, `signup` or `login` occurs.
```
Comment (mraerino) [inline README.md:294]: can we call this a webhook `receiver` or `target`?

File: README.md [hunk 2]
```diff
@@ -289,6 +289,27 @@ Default Content (if template is unavailable):
...
+
+`WEBHOOK_SECRET` - `string`
+
+Shared secret to authorize webhook requests.
```
Comment (mraerino) [inline README.md:298]: maybe add a note that this uses `JWS` signatures?

**Review by mraerino — COMMENTED** [review]
cool! just some ideas for improvement

File: README.md (hunk 1 above)
Comment (hydroid7) [reply README.md:294]: :+1:

File: README.md (hunk 2 above)
Comment (hydroid7) [reply README.md]: I extended the description a little bit, also to explain why JWS verification is important.

File: README.md [hunk 3]
```diff
@@ -291,11 +291,11 @@ Default Content (if template is unavailable):
 
 `WEBHOOK_URL` - `string`
 
-Url of the webhook listener server. This will called when events like  `validate`, `signup` or `login` occurs.
+Url of the webhook receiver server. This will called when events like  `validate`, `signup` or `login` occurs.
```
Comment (mraerino) [inline README.md:294]:
```suggestion
Url of the webhook receiver endpoint. This will be called when events like `validate`, `signup` or `login` occur.
```

**Review by mraerino — COMMENTED** [review]
cool. just one small suggestion left.

File: README.md [hunk 4]
```diff
@@ -289,6 +289,27 @@ Default Content (if template is unavailable):
 <p><a href="{{ .ConfirmationURL }}">Change Email</a></p>
 ```
 
+`WEBHOOK_URL` - `string`
+
+Url of the webhook receiver endpoint. This will called when events like `validate`, `signup` or `login` occur.
```
Comment (mraerino) [inline README.md:294]:
```suggestion
Url of the webhook receiver endpoint. This will be called when events like `validate`, `signup` or `login` occur.
```

**Review by mraerino — COMMENTED** [review]
sorry, you missed one 🙈

File: README.md (hunk 4 above)
Comment (hydroid7) [reply README.md]: Sure, I did it.

**Review by mraerino — APPROVED** [review]
thank you so much for the contribution!

**Pull request merged**